python/k4FWCore/ApplicationMgr.py

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright (c) 2014-2024 Key4hep-Project.
+#
+# This file is part of Key4hep.
+# See https://key4hep.github.io/key4hep-doc/ for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from Configurables import ApplicationMgr as AppMgr
+from Configurables import Reader, Writer, IOSvc, Gaudi__Sequencer
+import os
+from podio.root_io import Reader as PodioReader
+
+
+class ApplicationMgr:
+    """ApplicationMgr is a class that wraps the Gaudi ApplicationMgr class to
+    - Give the reader the collections it's going to read so that
```

Comment:
```suggestion

    - Give the reader the collections it's going to read so that
```

Need a new line before bullet lists in restructured text